app/src/main/java/ru/qveex/rst_tur/presentation/screens/home/HomeViewModel.kt

```diff
@@ -1,22 +1,26 @@
 package ru.qveex.rst_tur.presentation.screens.home
 
+import android.app.Application
 import android.util.Log
 import androidx.compose.runtime.mutableStateListOf
 import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
+import ru.qveex.rst_tur.*
 import ru.qveex.rst_tur.domain.interactors.Interactors
 import ru.qveex.rst_tur.domain.models.*
 import ru.qveex.rst_tur.utils.AppStatus
 import javax.inject.Inject
 
 @HiltViewModel
 class HomeViewModel @Inject constructor(
-    interactors: Interactors
-) : ViewModel() {
+    interactors: Interactors,
+    private val application: Application
+) : AndroidViewModel(application) {
```

Comment:
Using `AndroidViewModel` is really bad practice as is getting `application` and it's `context` in `viewModel's` classes
[AndroidViewModel Problematic for unit tests](https://stackoverflow.com/questions/44148966/androidviewmodel-vs-viewmodel#:~:text=AndroidViewModel%20Problematic%20for%20unit%20tests)

If you want get android resources in VM - you can create `ResourceProvider` (I usually use this solution) and inject it into vm